src/components/fieldset/_fieldset.scss

```diff
@@ -9,6 +9,16 @@
     padding: 0;
     border: 0;
     @include govuk-h-clearfix;
+
+    .govuk-c-input:last-of-type,
+    .govuk-c-select:last-of-type {
+      margin-bottom: 0;
+    }
+  }
+
+  .govuk-c-fieldset--error {
+    padding-left: $govuk-spacing-scale-3;
```

Comment:
In Elements, this is responsive - 10px on mobile, 15px on desktop. We don't have an equivalent in our responsive spacing scale, so I think we need to check that this behaviour is correct (unless you already have?)

Reply:
Good for @dashouse